src/nvim/runtime.h

```diff
@@ -44,6 +44,8 @@ enum {
   DIP_DIRFILE = 0x200,  ///< find both files and directories
 };
 
+typedef kvec_t(char *) CharVec;
```

Comment:
This was only present in `runtime.c` because it is used in a function prototype. In `stdpaths.c` it can just be `kvec_t(char *)` as it's local variable.

Reply:
aha okay.  is it not useful in somewhere else for future usage?

Reply:
In any case `runtime.h` is not a suitable header for it, because it has nothing to do with "runtime".

Reply:
okay thanks got it.

Reply:
could you please review